tests/sample_data/multimodel_statistics/test_multimodel.py

```diff
@@ -214,80 +196,202 @@ def multimodel_regression_test(cubes, span, name):
         raise RuntimeError(f'Wrote reference data to {filename.absolute()}')
 
 
-@pytest.mark.xfail(
-    raises=iris.exceptions.MergeError,
-    reason='https://github.com/ESMValGroup/ESMValCore/issues/956')
 @pytest.mark.use_sample_data
 @pytest.mark.parametrize('span', SPAN_PARAMS)
 def test_multimodel_regression_month(timeseries_cubes_month, span):
-    """Test statistic."""
+    """Test statistic fail due to differing input coordinates (pressure).
+
+    See https://github.com/ESMValGroup/ESMValCore/issues/956.
+
+    """
     cubes = timeseries_cubes_month
     name = 'timeseries_monthly'
-    multimodel_regression_test(
-        name=name,
-        span=span,
-        cubes=cubes,
+    msg = (
+        "Multi-model statistics failed to merge input cubes into a single "
+        "array"
     )
+    with pytest.raises(ValueError, match=msg):
+        multimodel_regression_test(name=name, span=span, cubes=cubes)
 
 
 @pytest.mark.use_sample_data
-@pytest.mark.parametrize('calendar', CALENDAR_PARAMS)
 @pytest.mark.parametrize('span', SPAN_PARAMS)
-def test_multimodel_regression_day(timeseries_cubes_day, span, calendar):
+def test_multimodel_regression_day_standard(timeseries_cubes_day, span):
+    """Test statistic."""
+    calendar = 'standard' if cf_units.__version__ >= '3.1' else 'gregorian'
```

Comment:
I think @bouweandela has changed them all to standard, probably zero chance to pick up the older cf_units

Reply:
This has been added recently in #1770: https://github.com/ESMValGroup/ESMValCore/pull/1770/files#diff-b4ea4c3198aae27d81beff94219ee691cee9c6611345ac05d384b580e389e7c9

So probably safe to keep

Reply:
safe it is then!